megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -4288,10 +4281,18 @@ else if ((atype != null)
         }
 
         // infantry squads are also hard to hit
-        if ((te != null) && te.isConventionalInfantry() && ((Infantry) te).isSquad()) {
+        if ((te instanceof Infantry) && te.isConventionalInfantry() && ((Infantry) te).isSquad()) {
             toHit.addModifier(1, Messages.getString("WeaponAttackAction.SquadTarget"));
         }
 
+        // pl-masc makes foot infantry harder to hit - IntOps p.84
+        if (te instanceof Infantry && te.hasAbility(OptionsConstants.MD_PL_MASC)
+                && te.getMovementMode() == EntityMovementMode.INF_LEG
```

Comment:
```suggestion
        if ((te instanceof Infantry) && te.hasAbility(OptionsConstants.MD_PL_MASC)
                && te.getMovementMode().isLegInfantry()
```

Our code style uses brackets when it comes to multiple operators.